utils/src/test/java/com/indix/gocd/utils/GoEnvironmentTest.java

```diff
@@ -53,4 +71,90 @@ public void shouldNotReplaceUnknownEnvVariables() {
 
         assertThat(replaced, is("COUNT:1 ${DOESNT_EXIST}"));
     }
+
+
+    @Test
```

Comment:
Is it just me or does these tests, test the workings of `BooleanUtils.toBooleanObject` ?